### ISSUE TITLE
Keyboard shortcut popup does not read out the message

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52025]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
Screenreader,NVDA[2018.2],MAS1.3.1

**Repro Steps:**
1.Launch VS Code while NVDA is ON.
2.Navigate to Settings icon in Activity Bar->Press Enter->Select "Keyboard Shortcuts" from dropdown.
3.Select any row and navigate to edit button and click on it.
4.Observed that screenreader is not reading the data present in the popup.

**Actual:**
NVDA is not reading the popup data "Pree desired key combination and then press Enter"
here NVDA is reading it as "code: AltLeft, keyCode: 18, key: Alt => UI: Alt+, user settings: alt+, dispatch: null edit blank"

**Expected:**
NVDA should reads the popup data.Here NVDA should reads "Pree desired key combination and then press Enter".

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
User will not understand what action have to performed.

**MAS Reference:**
MAS1.3.1- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

**Attachment For Reference:**
[[Accessibility]A11y_VSCode_KeyboardShortcuts_ScreenReader.pptx](https://github.com/Microsoft/vscode/files/2105899/Accessibility.A11y_VSCode_KeyboardShortcuts_ScreenReader.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This feels like law hanging fruit, the keyboard popup should simply read out the title text. Leaving to July
